deco-sites/otica-isabela
# Bugfix/pdp button name by category

## What is this contribution about?

Add button label by category within PDP

## How to test it?

Go to different PDPs based on the categories and check the label being different.
